<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1612128915133' stop='1612128934308'>
    <name>YandexInvalidLogin</name>
    <title>YandexInvalidLogin</title>
    <test-cases>
        <test-case start='1612128915134' status='failed' stop='1612128934308'>
            <name>YandexInvalidLogin</name>
            <title>YandexInvalidLogin</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1612128917046' status='passed' stop='1612128919298'>
                    <name>FailinputInLoginField</name>
                    <title>FailinputInLoginField</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Failed: Wait timed out after 15004ms</message>
                <stack-trace>TimeoutError: Wait timed out after 15004ms
                    at C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2201:17
                    at ManagedPromise.invokeCallback_ (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at runMicrotasks (&lt;anonymous>)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)
                From: Task: &lt;anonymous wait>
                    at scheduleWait (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2188:20)
                    at ControlFlow.wait (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2517:12)
                    at Driver.wait (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:934:29)
                    at run (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\built\browser.js:58:33)
                    at ProtractorBrowser.to.&lt;computed> [as wait] (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\built\browser.js:66:16)
                    at D:\Andersen\Project\JavaScript\sources\output_js\tests\invalid_login.js:67:63
                    at step (D:\Andersen\Project\JavaScript\sources\output_js\tests\invalid_login.js:33:23)
                    at Object.next (D:\Andersen\Project\JavaScript\sources\output_js\tests\invalid_login.js:14:53)
                    at fulfilled (D:\Andersen\Project\JavaScript\sources\output_js\tests\invalid_login.js:5:58)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)
                From: Task: Run it("YandexInvalidLogin") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (D:\Andersen\Project\JavaScript\sources\output_js\tests\invalid_login.js:45:5)
                    at addSpecsToSuite (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\anduser\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (D:\Andersen\Project\JavaScript\sources\output_js\tests\invalid_login.js:44:1)
                    at Module._compile (internal/modules/cjs/loader.js:1015:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
                    at Module.load (internal/modules/cjs/loader.js:879:32)
                    at Function.Module._load (internal/modules/cjs/loader.js:724:14)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>